client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -63,10 +80,11 @@ export const PluginPanel = ( {
 
 const mapStateToProps = state => {
 	const selectedSite = getSelectedSite( state );
+	const selectedSiteId = getSelectedSiteId( state );
```

Comment:
I think we could, but it seems more proper to me to grab it from the dedicated selector, while still being just as efficient (just accessing a node on an object).